package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "material-ui-search-bar",
-  "version": "1.0.0-beta.12",
+  "version": "1.0.0-beta.13",
```

Comment:
Please let us bump the version. We're using [np](https://www.npmjs.com/package/np) to release, it gets confused otherwise.